src/posit/connect/resources.py

```diff
@@ -44,31 +44,7 @@ def __setattr__(self, name: str, value: Any) -> None:
         raise AttributeError("cannot set attributes: use update() instead")
 
 
-class Resources(ABC, Generic[T]):
-    @abstractmethod
-    def create(self, *args, **kwargs) -> T:
-        raise NotImplementedError()
-
-    @abstractmethod
-    def delete(self, *args, **kwargs) -> None:
-        raise NotImplementedError()
-
-    @abstractmethod
-    def find(self, *args, **kwargs) -> List[T]:
-        raise NotImplementedError()
-
-    @abstractmethod
-    def find_one(self, *args, **kwargs) -> Optional[T]:
-        raise NotImplementedError()
-
-    @abstractmethod
-    def get(self, *args, **kwargs) -> T:
-        raise NotImplementedError()
-
-    @abstractmethod
-    def update(self, *args, **kwargs) -> T:
-        raise NotImplementedError()
-
-    @abstractmethod
-    def count(self, *args, **kwargs) -> int:
-        raise NotImplementedError()
```

Comment:
I can cherry-pick this out if needed. I'm realizing that not all API endpoints can support every one of these methods.

Reply:
Fine by me to leave it here (and I'm always happy about deleting code 😂 )